### DISCUSSION TITLE
2.3.10 osquery windows_events pull failure after autoupdate

### DISCUSSION THREAD

**Discussion by 0nnyx** [Security-Onion-Solutions/securityonion#2134] [Unsupported Versions]
Using 2.3.10 ISO and preconfigured launcher.msi packages, windows_events were pulled fine using `SELECT * FROM windows_events;"` every 3 minutes until launcher and osquery got autoupdated on Nov 25 : Found launcher.exe-updates and osqueryd.exe-updates with timestamp matching the time that Kibana has no more sysmon/windows logs.
After uninstall & clearing the Kolide directory, then reinstalling the prepackaged launcher.msi, the windows_events are pulled properly again.
By pull failure, I mean the query is sent without failure from Fleet but returns with an empty result.
Note: other packs checking for scheduled tasks & auto start services were still working after the autoupdate, only the windows evtx one was failing.

**Comment by 0nnyx** [comment]
After reinstalling and having launcher & osquery autoupdated, windows logs are now properly gathered. 
Will see if that problem happens again. 2 out of 4 windows computers were not getting their logs pulled for days after that autoupdate.
Never had such problem with winlogbeat.

**Reply by defensivedepth** [reply]
If you see this again, please review Launcher logs in the `Application`  eventchannel - That should give some insight into what is going on, as that should not be happening.

**Reply by 0nnyx** [reply]
About the Application event logs, is there any way to quiet down Launcher to the essential ? It's really polluting the logs each time it can't execute a command like `Failed to retrieve network statistics for interface,  Failed to expand globs: Failed to open registry handle, Failed to get cwd , Failed to get PEB UPP`. I tried setting the flag option `logger_stderr=false` but it didn't improve.

**Reply by defensivedepth** [reply]
@0nnyx Are you still seeing these Launcher logs? Should have been cleaned up more throughout the previous releases.
